src/Apps/ViewingRoom/routes.tsx

```diff
@@ -0,0 +1,21 @@
+import loadable from "@loadable/component"
+import { graphql } from "react-relay"
+
+const ViewingRoomApp = loadable(() => import("./ViewingRoomApp"))
+
+export const routes = [
+  {
+    path: "/viewing-room",
```

Comment:
[note for later] We'll eventually want a `/viewing-room/:id/works` route too.

Reply:
Ah, nice catch, will update 👍